packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/ViewDiagramDescriptionConverter.java

```diff
@@ -223,16 +229,30 @@ private NodeDescription convert(org.eclipse.sirius.components.view.NodeDescripti
             return this.stylesFactory.createNodeStyle(effectiveStyle, optionalEditingContextId);
         };
 
+        Function<VariableManager, ILayoutStrategy> childrenLayoutStrategyProvider = variableManager -> {
+            ILayoutStrategy childrenLayoutStrategy = null;
+
+            LayoutStrategyDescription childrenLayoutStrategyFromViewModel = viewNodeDescription.getChildrenLayoutStrategy();
+
+            if (childrenLayoutStrategyFromViewModel instanceof ListLayoutStrategyDescription) {
+                childrenLayoutStrategy = ListLayoutStrategy.newListLayoutStrategy().direction(LayoutDirection.COLUMN).build();
```

Comment:
I know that currently we have a single possible direction, but `ListLayoutStrategyDescription` specifies it, so it would be more future-proof to take the value from there than to hard-code `direction(LayoutDirection.COLUMN)`.

Reply:
We have removed the direction concept for now. So ListLayoutStrategyDescription does not have any feature anymore